AirportDepartures.playground/Contents.swift

```diff
@@ -58,9 +153,29 @@ import UIKit
 //:     Destination: Los Angeles Airline: Delta Air Lines Flight: KL 6966 Departure Time:  Terminal: 4 Status: Canceled
 //:     Destination: Rochester Airline: Jet Blue Airways Flight: B6 586 Departure Time: 1:26 PM Terminal:  Status: Scheduled
 //:     Destination: Boston Airline: KLM Flight: KL 6966 Departure Time: 1:26 PM Terminal: 4 Status: Scheduled
-
-
-
+func printDepartures2(departureBoard: DepartureBoard) {
+    print("Departures for \(jfkDepartures.airport.abbreviation) - \(jfkDepartures.airport.name) in \(jfkDepartures.airport.location)")
+    for flight in jfkDepartures.departures {
+        if flight.departureTime == nil && flight.terminal == nil {
+            print("Destination: \(flight.destination.name) - \(flight.destination.abbreviation)  Airline: \(flight.airline)  Flight #: \(flight.number)  Departure Time:  Terminal:  Status: \(flight.status.0.rawValue)\(flight.status.1?.rawValue ?? "")")
+        } else if flight.departureTime != nil && flight.terminal == nil {
+            if let time = flight.departureTime {
+                print("Destination: \(flight.destination.name) - \(flight.destination.abbreviation)  Airline: \(flight.airline)  Flight #: \(flight.number)  Departure Time: \(time)  Terminal:  Status: \(flight.status.0.rawValue)\(flight.status.1?.rawValue ?? "")")
+            }
+        } else if flight.departureTime == nil && flight.terminal != nil {
+            if let terminal = flight.terminal {
+                print("Destination: \(flight.destination.name) - \(flight.destination.abbreviation)  Airline: \(flight.airline)  Flight #: \(flight.number)  Departure Time:  Terminal: \(terminal)  Status: \(flight.status.0.rawValue)\(flight.status.1?.rawValue ?? "")")
+            }
+        } else if let time = flight.departureTime {
+            if let terminal = flight.terminal {
+                print("Destination: \(flight.destination.name) - \(flight.destination.abbreviation)  Airline: \(flight.airline)  Flight #: \(flight.number)  Departure Time: \(time)  Terminal: \(terminal)  Status: \(flight.status.0.rawValue)\(flight.status.1?.rawValue ?? "")")
+            }
+        }
+    }
+}
```

Comment:
I would challenge you to see if you can do this with only one print statement. One way to approach it would be to set up a variable to a default value, and if you can unwrap your optional, set that variable to the unwrapped value.

``` swift
var terminal = ""
if let flightTerminal = flight.terminal {
    terminal = flightTerminal
}

// terminal is now equal to either an empty string "" or the unwrapped flight terminal
```